### PR TITLE
11852 ios consistent star rating

### PR DESCRIPTION
**Pull request by adam-govan**
https://swrvedev.jira.com/browse/SWRVE-11852

@dominicmarmionswrve / @Sergio-Mira

### CODE REVIEW

File: Sdk/Conversation/SwrveConversationItemViewController.m [hunk 1]
```diff
@@ -52,6 +52,7 @@ - (void)viewDidAppear:(BOOL)animated
 
 -(void) viewWillAppear:(BOOL)animated {
     [super viewWillAppear:animated];
+    [self.contentTableView setHidden:YES];
```
Comment (Sergio-Mira) [inline Sdk/Conversation/SwrveConversationItemViewController.m]: Why does it need to be hidden? Doesn't this mean that no atom will be visible until all are loaded?

Comment (adam-govan) [inline Sdk/Conversation/SwrveConversationItemViewController.m]: that's exactly right, It hides the jumps in the Text/ Star Rating. The view will be hidden for second, but the jump is far worse looking

Comment (Sergio-Mira) [inline Sdk/Conversation/SwrveConversationItemViewController.m]: The jumps happen in all platforms, has there been a request to change it?

Comment (adam-govan) [inline Sdk/Conversation/SwrveConversationItemViewController.m]: yes,
https://swrvedev.jira.com/browse/SWRVE-11815

**Comment by Sergio-Mira** [comment]
:rocket:

**Pull request merged**